src/components/DatePicker/index.js

```diff
@@ -1,82 +1,63 @@
-import React, { Component } from 'react';
-import { DatePicker } from 'antd';
+import { Split, SplitItem, DatePicker, isValidDate, yyyyMMddFormat } from '@patternfly/react-core';
+import React, { useState } from 'react';
 import { connect } from 'dva';
-import moment from 'moment';
-import PropTypes from 'prop-types';
-import './index.less';
 
-const { RangePicker } = DatePicker;
+const mapStateToProps = state => {
+  return state;
+};
 
-@connect(({ global, datastore }) => ({
-  indices: datastore.indices,
-  selectedDateRange: global.selectedDateRange,
-}))
-class AntdDatePicker extends Component {
-  static propTypes = {
-    onChangeCallback: PropTypes.func,
-  };
+const PFDatePicker = props => {
+  const [from, setFrom] = useState('');
+  const [to, setTo] = useState('');
+
+  const toValidator = date =>
+    isValidDate(from) && date >= from ? '' : '"From" date must be less than or equal to "To" date';
 
-  static defaultProps = {
-    onChangeCallback: () => {},
+  const onFromChange = (_str, date) => {
+    setFrom(date);
   };
 
-  handleChange = (dates, dateStrings) => {
-    const { dispatch, onChangeCallback } = this.props;
+  const getDataOnChange = (fromDate, toDate) => {
+    const { dispatch, onChangeCallback } = props;
```

Comment:
this line can move to the top of the function component scope

Reply:
fixed!